buildSrc/src/main/java/Dependencies.kt

```diff
@@ -0,0 +1,69 @@
+object Versions {
+
+    // Build Config
+    const val minSDK = 19
+    const val compileSDK = 29
+    const val targetSDK = 29
+    const val buildToolsVersion = "29.0.2"
+
+    // App version
+    const val appVersionCode = 1
+    const val appVersionName = "1.0.0"
+
+    // Plugins
+    const val androidGradlePlugin = "3.5.2"
+
+    // Kotlin
+    const val kotlin = "1.3.60"
+    const val jvmTarget = "1.8"
+
+    // Androidx
+    const val support = "1.1.0"
+    const val constraintLayout = "1.1.0"
+
+    // Testing
+    const val junit = "4.12"
+    const val espresso = "3.2.0"
+    const val testJunit = "1.1.1"
+    const val robolectric = "4.3-beta-1"
+    const val coroutinesTest = "1.2.1"
+
+    //Retrofit
+    const val retrofit = "2.6.2"
+
+    //Java
+    const val javaCompatibility = "7"
+
+    //Detekt
+    const val detekt  = "1.1.1"
+}
+
+object Deps {
```

Comment:
Please rename this as `dependencies` to improve understanding, also, please check if you can create different objects for the different types of dependencies.

Reply:
the dependencies are separated into different objects